app/routes/errors.py

```diff
@@ -19,7 +19,7 @@
 from app.views.handlers.confirmation_email import (
     ConfirmationEmailFulfilmentRequestPublicationFailed,
 )
-from app.views.handlers.feedback import FeedbackLimitReached
+from app.views.handlers.feedback import FeedbackLimitReached, FeedbackUploadFailed
 from app.views.handlers.individual_response import (
```

Comment:
```suggestion
from app.views.handlers import (
```

The `IndividualResponseFulfilmentRequestPublicationFailed`and `IndividualResponseLimitReached` need to be added to the handlers init file for the above to work

Reply:
eh?